build

```diff
@@ -33,10 +33,12 @@ case "${RKT_STAGE1_USR_FROM}" in
 esac
 
 echo "Building rkt (stage0)..."
-go build -o ${GOBIN}/rkt \
+go build -o ${GOBIN}/rkt -v \
 	${RKT_STAGE1_IMAGE:+-ldflags "-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'"} \
 	${REPO_PATH}/rkt
 
+[[ $? == 0 ]] || echo "build failed"; exit
```

Comment:
@jonboulle Yes, you are correct, I'll drop the error check patch.  I'll work on the arm64 build problem today and post an update when ready.